srcpkgs/qbe/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'qbe'
+pkgname=qbe
+version=1.0
+revision=1
+archs="x86_64* aarch64*"
```

Comment:
Doesn't it run on all architectures but only can build for those you listed?

Reply:
You're right. I updated the PR.